packages/inflection-tables/tests/data/greek-adjective-paradigm.test.js

```diff
@@ -31,15 +31,15 @@ describe('greek-adjective-paradigm.test.js', () => {
 
     expect(inflectionsViewSet.hasMatchingViews).toBeTruthy()
 
-    expect(inflectionsViewSet.matchingViews.length).toEqual(2)
+    expect(inflectionsViewSet.matchingViews.length).toBeGreaterThanOrEqual(4)
```

Comment:
why not test for exact equality on number of views?

Reply:
Oh, I only simplified tests for me - but if you think - it is important - I will fix it

Reply:
thanks! I think it's good to have an exact test because it can help us find when we've done something that changes it.